### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by pwalczysko**
Similar to https://github.com/ome/ansible-role-basedeps/pull/11

cc @sbesson @jburel

### CODE REVIEW

**Review by sbesson — APPROVED** [review]

**Comment by pwalczysko** [comment]
Although the changes are similar to the other repos like https://github.com/ome/ansible-role-ice/pull/21, there is some (yet inexplicable for me) failure of the test assertion. I will investigate after AL.

**Comment by pwalczysko** [comment]
@sbesson this is green now, also removed the ubuntu1804 as in the other repos. Please merge if happy.

File: meta/main.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ galaxy_info:
     - name: Ubuntu
       versions:
         - bionic
```
Comment (sbesson) [inline meta/main.yml:14]: Maybe let's remove this line from the metadata file to indicate support for Ubuntu 18.04 is dropped

**Review by sbesson — CHANGES_REQUESTED** [review]
One suggestion to match the Ubuntu 18.04 support removal. Otherwise good to merge

File: meta/main.yml (hunk 1 above)
Comment (sbesson) [reply meta/main.yml]: Sorry, I actually meant the whole `Ubuntu` block

**Comment by pwalczysko** [comment]
> Sorry, I actually meant the whole `Ubuntu` block

This is now fixed, thanks.

**Comment by pwalczysko** [comment]
Do we want to merge @jburel please ?

**Pull request merged**